test/extensions/filters/network/wasm/config_test.cc

```diff
@@ -128,304 +130,6 @@ TEST_P(WasmNetworkFilterConfigTest, YamlLoadInlineBadCode) {
                             "Failed to initialize WASM code from <inline>");
 }
 
-TEST_P(WasmNetworkFilterConfigTest, YamlLoadFromRemoteWASM) {
```

Comment:
Note: these tests are redundant with the ones in filters/http and because Envoy has 2 types of test time which are mutually incompatible and which are used by MockDispatcher and by MockConnection these tests can't run here until that is fixed.  Envoy has an outstanding issue on this.